mucommander-preferences/src/main/java/com/mucommander/conf/MuPreferences.java

```diff
@@ -232,7 +235,7 @@ public class MuPreferences implements MuPreferencesAPI {
 	/** Default value for 'Quick search timeout' */
 	public static final int DEFAULT_QUICK_SEARCH_TIMEOUT           = 2;
 
-	/** Name of the root element's attribute that contains the version of muCommander used to write the CONFIGURATION file. */
+    /** Name of the root element's attribute that contains the version of muCommander used to write the CONFIGURATION file. */
 	static final String VERSION_ATTRIBUTE = "version";
```

Comment:
something about the indentation here seems wrong

Reply:
yup, because this files has tabs as indents. I will make it look ok, won't convert it to spaces tho.